src/mgos_rpc.c

```diff
@@ -598,6 +598,9 @@ static void mgos_rpc_call_oplya(struct mg_rpc *c, void *cb_arg,
                                 struct mg_str error_msg) {
   struct mgos_rpc_call_eh_data *oplya_arg =
       (struct mgos_rpc_call_eh_data *) cb_arg;
+  if (oplya_arg->cb == NULL) {
```

Comment:
we can perform this check earlier, in mgos_rpc_call, and not provide callback to mg_rpc_callf at all

Reply:
Done.